coffee/chosen.jquery.coffee

```diff
@@ -179,10 +187,8 @@ class Chosen extends AbstractChosen
     @container.addClass "chosen-container-active"
     @active_field = true
 
-    @search_field.val(@search_field.val())
```

Comment:
This was actually a difference between the jQuery and Prototype versions — the Prototype version seen below is also removed, but it was: 

```coffeescript
@search_field.value = this.get_search_field_value()
```

That’s _almost_ the same, but not quite — resulting in the initial keypress being doubled in the Prototype version due to the change in that method (for example, keyboard focusing the element and pressing <kbd>R</kbd> resulted in `RR` in Prototype but `R` in jQuery). 

I couldn’t see a reason for setting the value of the field to… itself… so I dropped it completely to make the behavior consistent. 

Looking at the git history, it appears to originate in [465eb61](https://github.com/harvesthq/chosen/blame/465eb61eb690d01d2495e33a2b85f1000a0f1f6d/coffee/chosen.jquery.coffee#L182), the initial commit by @pfiller, humorously mentioning “Plenty of bugs at this point!”